internal/operators/odf/validations.go

```diff
@@ -154,24 +155,24 @@ func validateRequirements(o *operator, odfClusterResources *odfClusterResourcesI
 }
 
 func (o *operator) setStatusInsufficientResources(odfClusterResources *odfClusterResourcesInfo, mode odfDeploymentMode) string {
-	status := fmt.Sprint("Insufficient Resources to deploy ODF in ", string(mode), " Mode. ")
+	status := fmt.Sprint("Insufficient resources to deploy ODF in ", string(mode), " mode. ")
```

Comment:
Just use https://www.geeksforgeeks.org/how-to-convert-a-string-in-lower-case-in-golang/ here instead of having the caller use lowercase just for the message